addons/website/static/src/js/content/website_root.js

```diff
@@ -282,8 +282,8 @@ var WebsiteRoot = BodyManager.extend({
         var $buttons = $(ev.currentTarget).find('button[type="submit"], a.a-submit');
         _.each($buttons, function (btn) {
             var $btn = $(btn);
-            $btn.attr('data-loading-text', '<i class="fa fa-spinner fa-spin"></i> ' + $(btn).text());
-            $btn.button('loading');
+            $btn.html('<i class="fa fa-spinner fa-spin"></i> ' + $(btn).text());
+            $btn.attr('disabled', true);
```

Comment:
`prop` instead of `attr` :wink:
No need of writing the DOM and "true" would not be a valid value for that attribute anyway ;)